src/termination.rs

```diff
@@ -23,19 +23,13 @@ pub struct TerminationConfig {
 impl Lockup {
     pub fn terminate(
         &mut self,
-        initiator_id: &AccountId,
         hashed_schedule: Option<Schedule>,
         termination_timestamp: TimestampSec,
-    ) -> Balance {
+    ) -> (Balance, AccountId) {
         let termination_config = self
             .termination_config
             .take()
             .expect("No termination config");
-        assert_eq!(
-            termination_config.beneficiary_id.as_ref(),
-            initiator_id,
-            "Unauthorized"
-        );
```

Comment:
@evgenykuzyakov We decided that any "admin" (a.k.a. "deposit whitelisted user") can terminate lockups, but the funds will be refunded back to the original depositor (be it DAO or some independent account). Is it fine with your use-cases?